pinot-core/src/main/java/org/apache/pinot/core/operator/filter/OrFilterOperator.java

```diff
@@ -32,6 +34,8 @@
   private final List<BaseFilterOperator> _filterOperators;
   private final int _numDocs;
 
+  private boolean _anyChildrenMatchAll = false;
```

Comment:
Same comment as the `AndFilterOperator`. All the children cannot have `isResultMatchingAll()` or `isResultEmpty()` as `false`

Reply:
This has been removed for now.